tests/unit/neptune/new/test_stringify_unsupported.py

```diff
@@ -44,6 +44,12 @@ class Obj:
     def __init__(self, name: str = "A"):
         self._name = name
 
+    def __len__(self) -> int:
```

Comment:
Use a list of objects instead in tests. `Obj` should be just a single value and really similar to dataclass (or even we can made it a dataclass, but no need to do to my knowledge)